### PR TITLE
adds links to every audit rule in readme

### PR DESCRIPTION
**Pull request by jaimeiniesta**
This commit updates the README.md file to add direct links to every audit rule's description on the ADT wiki.

### CODE REVIEW

**Comment by ckundo** [comment]
Does this need to wait for #12 to be closed?

**Comment by jaimeiniesta** [comment]
I would merge this first, and then I'll update the links after #12

File: README.md [hunk 1]
```diff
@@ -53,27 +53,45 @@ Run the audit from a Ruby application like so
...
+[AX_FOCUS_03][AX_FOCUS_03] | Avoid positive integer values for tabIndex.
+[AX_HTML_01][AX_HTML_01]   | The web page should have the content's human language indicated in the markup.
+[AX_IMAGE_01][AX_IMAGE_01] | Meaningful images should not be used in element backgrounds.
+[AX_TEXT_01][AX_TEXT_01]   | Controls and media elements should have labels.
+[AX_TEXT_02][AX_TEXT_02]   | Images should have an alt attribute.
+[AX_TEXT_04][AX_TEXT_04]   | The purpose of each link should be clear from the link text.
+[AX_TITLE_01][AX_TITLE_01] | The web page should have a title that describes topic or purpose.
+[AX_VIDEO_01][AX_VIDEO_01] | Video elements should use <track> elements to provide captions.
+
+[AX_ARIA_01]: https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_01
```
Comment (ckundo) [inline README.md]: I think we want these to be clickable. Will the urls turn into links automatically?

Comment (jaimeiniesta) [inline README.md]: They will, that's the way to define links as footnotes. At least, markdown preview in Atom makes them clickable.

Comment (jaimeiniesta) [inline README.md]: See it working here: https://github.com/jaimeiniesta/access_lint/tree/patch-2

Comment (ckundo) [inline README.md]: Oh cool! TIL :)

**Pull request merged**